node/src/components/consensus/highway_core/finality_detector.rs

```diff
@@ -171,12 +172,12 @@ mod tests {
         // b0: 10           b1: 4
         //        \
         //          c0: 1 — c1: 1
-        let b0 = add_vote!(state, rng, BOB, 0xB0; N, N, N)?;
-        let c0 = add_vote!(state, rng, CAROL, 0xC0; N, b0, N)?;
-        let c1 = add_vote!(state, rng, CAROL, 0xC1; N, b0, c0)?;
-        let a0 = add_vote!(state, rng, ALICE, 0xA0; N, b0, N)?;
-        let a1 = add_vote!(state, rng, ALICE, 0xA1; a0, b0, c1)?;
-        let b1 = add_vote!(state, rng, BOB, 0xB1; a0, b0, N)?;
+        let b0 = add_vote!(state, instance_id, rng, BOB, 0xB0; N, N, N)?;
```

Comment:
Almost none of the tests will actually deal with different instance IDs. Can we just fix using the default test ID in the macro instead of passing it in in every line?

The one test where we want to use the wrong one should just create the `WireVote` directly.